### PR TITLE
Added frontend CI

### PR DESCRIPTION
**Pull request by Sahil1479**

### CODE REVIEW

File: .github/workflows/frontend_CI.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        node-version: ${{ matrix.node-version }}
+    
+    - name: Install dependencies
+      working-directory: ./frontend
+      run: npm install
+    
+    - name: Format
+      working-directory: ./frontend/src
+      run: |
+          npm ci
```
Comment (Tanuj22) [inline .github/workflows/frontend_CI.yml:30]: Why this?

File: .github/workflows/frontend_CI.yml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+          npm run format
```
Comment (Tanuj22) [inline .github/workflows/frontend_CI.yml:31]: Remove this. We only want to check and not format.

Comment (Sahil1479) [reply .github/workflows/frontend_CI.yml:31]: ok, I will remove it.

**Comment by Tanuj22** [comment]
Remove these files
```
[warn] .eslintrc
[warn] .prettierrc
[warn] .vscode/settings.json
[warn] README.md
``` 
from prettier by adding them to a `..prettierignore`  file

**Pull request closed without merging**